CalcualtorLogic.js

```diff
@@ -173,8 +165,8 @@ class Calculator {
             if (this.operator === "*") _result = calculatorLogic.multiply(firstOperand, secondOperand);
```

Comment:
I can see `calculatorLogic` is defined as property on the `Calculator` class 
also `currentResult` is defined as property on the same class 

Why when you want to refrence `currentResult` you are doing with using the `this` keyword
but you're not doing that with `calculatorLogic`?
why you're trying to access the `calculatorLogic` directly without using the `this` keyword